ui/components/contact-email.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Link from './link';
+
+export const email = 'PolicyLibrary@omb.eop.gov';
+const mailto = `mailto:${email}`;
+const mailtext = `Email ${email}`;
+
+/* Has the same href value every time, making the global email address easy to change. */
+export function ContactEmail({ text }) {
```

Comment:
Something to consider in terms of interface design -- do we want to use React children instead of a prop? If we did, we could call:
```jsx
<ContactEmail>contact us</ContactEmail>
```
It'd also let us use non-strings:
```jsx
<ContactEmail><img src="mailbox.png" /></ContactEmail>
```
Getting the default value would be a bit more complicated -- I'm not sure we could use `defaultProps`.

Reply:
I went with the easy option here, because I wasn't sure how to handle the default value, but this seems like a good enhancement. I might leave it until we hit a point where we know we want something that's not flat text, though.